### DISCUSSION TITLE
install/release-notes/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1689] [Announcements]
# install/release-notes/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/release-notes/

**Comment by jzhupup** [comment]
支持外接 Redis 组件，但是目前容器管理还没适配

**Reply by windsonsea** [reply]
这个评论稍有错位，先 close 掉了额
